sensu/config.go

```diff
@@ -72,6 +73,19 @@ func NewConfigFromFile(flagset *configFlagSet, configFile string) (*Config, erro
 		return nil, errNoClientName
 	}
 
+	// If we reached this point, we have a client name.
+	// Let's also add the default subscription - client:name
+	// Without at least one subscription sensu server will crash.
+	cfg.config.Client.Subscriptions = append(
```

Comment:
do you really need to re assign  `cfg.config.Client.Subscriptions` twice?

you could do both at the same time:

```golang
cfg.config.Client.Subscriptions = removeDuplicates(
    append(
        cfg.config.Client.Subscriptions, 
        fmt.Sprintf("client:%s", cfg.config.Client.Name),
    ),
)
```